### DISCUSSION TITLE
Include a down function in the generated settings migration

### DISCUSSION THREAD

**Discussion by CapitaineToinon** [spatie/laravel-settings#297] [Ideas]
Hey,

I just think it would be great if the generated migration file by `make:settings-migration` included a `down` method as well. I often do database refreshes when working locally and I've forgotten to add the down migration myself a couple of times, yet I still use the migration command as it generates the file in the correct location with a correct timestamp in the file and all.

That's it. From what I can tell from the source code that would be just updating a string so I can do the PR.

Let me know.

**Comment by rubenvanassche** [comment]
We don't use down migrations in any package / project.

99.9% of the time you are not gonna use a down migration and you’re probably never testing them. Just write an up migration if you want to drop a column again. 👍
